CharacterCreation/02_Classes.txt

```diff
@@ -388,9 +388,15 @@ Skill Proficiencies:
 
 
 =====   HUNTER   =====
-	An expert tracker, you're the one hired when someone wants something found. 
-Expertly trained in manipulating your Nanites to aid in pursuit of a quarry, or 
-in marking enemies for execution.
+
+	Unlike many of your compatriots who prefer the cold comforts of technology,
+you have an affinity for the frontier. The great outdoors. Be they the 
+wind-swept plains of frontier farms, the thick jungles of Tundaar or the swamps
+of Trioch you love it all. You are much more likely to know which plants won't 
```

Comment:
I think there should be some punctuation here: of Trioch; you love it all.

Reply:
Semicolon is a bit of overkill, but a comma or dash would add to the text.